release-notes/opensearch-release-notes-2.17.0.md

```diff
@@ -0,0 +1,925 @@
+# OpenSearch and OpenSearch Dashboards 2.17.0 Release Notes
+
+## Release Highlights
+
+OpenSearch 2.17 includes new and updated features to help you build and optimize your search applications, improve stability, availability, and resiliency, enhance ease of use, and more. 
+
+### NEW AND UPDATED FEATURES
+
+* Introduced as an experimental feature in OpenSearch 2.15, Remote Cluster State Publication is now generally available in 2.17.
```

Comment:
no need to capitalize remote cluster state publication. We do not capitalize in docs. https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/remote-store/remote-cluster-state/